docs/guides/jobs/installing_dependencies.md

```diff
@@ -4,7 +4,7 @@ If you need additional or custom packages, you can install them before running y
 
 ## Installing Python dependencies
 
-If your code needs additional Python packages at run time, you can add them to a special file named `floyd_requirements.txt`. 
+If your code needs additional Python packages at run time, you can add them to a special file named `floyd_requirements.txt`.
```

Comment:
Add "if you want to install non-Python packages, please see here" at the end

Reply:
Done